torch/csrc/jit/ir/ir.cpp

```diff
@@ -1897,6 +1897,19 @@ std::vector<Value*> inlineCallTo(
   std::unordered_map<InlinedCallStack*, InlinedCallStackPtr>
       new_callstack_entries;
 
+  c10::optional<ModuleInstanceInfo> module_instance_info = c10::nullopt;
+  if (to_replace->kind() == prim::CallMethod) {
```

Comment:
I think CallFunction nodes are like free functions. CallMethods are methods on some objects and we get the objects via GetAttr nodes. If it is a CallFunction it can be any function. That function maybe called from some method (CallMethod) of some object but that information we will have captured when we were inlining the node corresponding to that method.
See: https://github.com/pytorch/pytorch/blob/master/torch/csrc/jit/runtime/interpreter.cpp#L930-L940